database/metadata/databases/default/tables/public_crashes_list_view.yaml

```diff
@@ -197,5 +197,4 @@ select_permissions:
         - rpt_street_sfx
         - crash_timestamp
       filter: {}
-      allow_aggregations: true
```

Comment:
i think this change is causing the location details page to bomb. i am seeing a failing network request with this error:
```
{
    "errors": [
        {
            "message": "field 'crashes_list_view_aggregate' not found in type: 'query_root'",
            "extensions": {
                "path": "$.selectionSet.crashes_list_view_aggregate",
                "code": "validation-failed"
            }
        }
    ]
}
```

Reply:
weird, the locations page isnt bombing out for me. and its odd that we only had aggregations allowed for vz-admin and not for any other user types?

Reply:
ah, ok. i tested this with an editor user, and the query is still failing but my editor is not bombing. this is causing the little widgets to not render on non-admin pages. so this may be a preexisting bug:

<img width="820" alt="Screenshot 2024-10-23 at 3 55 41 PM" src="https://github.com/user-attachments/assets/e3bb2bf0-3ab6-4cd2-ab08-0fd35690ce11">

<img width="541" alt="Screenshot 2024-10-23 at 3 57 36 PM" src="https://github.com/user-attachments/assets/a5ab416f-83ca-452e-8b6e-4f2184e2b69a">

enabling aggregate query permissions for all users times on the crashes_list_view should fix this 👍